lms/djangoapps/verify_student/views.py

```diff
@@ -1074,14 +1075,6 @@ def results_callback(request):  # lint-amnesty, pylint: disable=too-many-stateme
         "Date": request.META.get("HTTP_DATE", "")
     }
 
-    has_valid_signature(
```

Comment:
Need to remove this from the imports as well

Reply:
Done! Thank you.